<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ICHCTimeFrame</class>
 <widget class="QFrame" name="ICHCTimeFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Vertical</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="verticalineEdit"/>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Grip/put-off 1</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="fixture1GripLineEdit"/>
   </item>
   <item row="0" column="6">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="fixture1PutOffLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Horizontal</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="horizontalLineEdit"/>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Grip/put-off 2</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="fixture2GripLineEdit"/>
   </item>
   <item row="1" column="6">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="fixture2PutOffLineEdit"/>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Grip/put-off 3</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="fixture3GripLineEdit"/>
   </item>
   <item row="2" column="6">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="fixture3PutOffLineEdit"/>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Grip/put-off 4</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="fixture4GripLineEdit"/>
   </item>
   <item row="3" column="6">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="fixture4PutOffLineEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Secondary arm up</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="secondaryArmUpLineEdit"/>
   </item>
   <item row="4" column="4">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Suck-in/put-off 1</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="sucker1SuckInLineEdit"/>
   </item>
   <item row="4" column="6">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="sucker1PutOffLineEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Secondary arm down</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="secondaryArmDownLineEdit"/>
   </item>
   <item row="5" column="4">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Suck-in/put-off 2</string>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="sucker2SuckInLineEdit"/>
   </item>
   <item row="5" column="6">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="sucker2PutOffLineEdit"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Secondary arm go forward</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="secondayGoForwardLineEdit"/>
   </item>
   <item row="6" column="4">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Injection</string>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="injectionOpenLineEdit"/>
   </item>
   <item row="6" column="6">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="injectionCloseLineEdit"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Secondary arm go backward</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="secondayGoBackwardLineEdit"/>
   </item>
   <item row="7" column="4">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Conveyor</string>
     </property>
    </widget>
   </item>
   <item row="7" column="5">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="conveyorOpenLineEdit"/>
   </item>
   <item row="7" column="6">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="conveyorCloseLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>Main Arm Forward</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>Main Arm Backward</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="mainArmForwardEdit"/>
   </item>
   <item row="3" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="mainArmBackwardEdit"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_18">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_19">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_20">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label_21">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="label_22">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="label_23">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLabel" name="label_24">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QLabel" name="label_25">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QLabel" name="label_26">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QLabel" name="label_27">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="3" column="7">
    <widget class="QLabel" name="label_28">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="4" column="7">
    <widget class="QLabel" name="label_29">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="5" column="7">
    <widget class="QLabel" name="label_30">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="6" column="7">
    <widget class="QLabel" name="label_31">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="7" column="7">
    <widget class="QLabel" name="label_32">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ICLineEditWithVirtualNumericKeypad</class>
   <extends>QLineEdit</extends>
   <header>iclineeditwithvirtualnumerickeypad.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
